src/api/exportable-lib/index.js

```diff
@@ -4,6 +4,7 @@ const SelectorBuilder       = lazyRequire('../../client-functions/selectors/sele
 const role                  = lazyRequire('../../role');
 const createRequestLogger   = lazyRequire('../request-hooks/request-logger');
 const createRequestMock     = lazyRequire('../request-hooks/request-mock');
+const browserProviderPool   = lazyRequire('../../browser/provider/pool');
```

Comment:
`exportable-lib` should contain only Test API. Please extract `list-browsers` to the `/src/utils/list-browsers` and export it through [/src/embedding-utils](https://github.com/DevExpress/testcafe/blob/master/src/embedding-utils.js)